README.md

```diff
@@ -27,16 +27,35 @@ This makes it possible to separate away the escrow coordinator from the trading
 Both trading parties have to commit to their trade obligations to the coordinator. This commitment has to contain all information necessary for the coordinator to decide which trade party fulfilled their obligations in the case of an escrow mediation. This can include payout information, amounts, timeframes and a freely written trade contract. When possible, information can be submitted as hash to improve privacy against the coordinator.
 
 ##### Nostr communication
-To reduce uneccesary burden on relays we can aim to use ephemeral event types for communication between traders and coordinator.
+To reduce unnecessary burden on relays we can aim to use ephemeral event types for communication between traders and coordinator.
 
 ##### Client
 The client could be distributed as wasm library and rust crate. There could also be a compilation flag that decides if the client gets built with nostr communication logic or only with nostr event creation logic. First would be useful for inclusion in traditional trading platforms and second would be useful for nostr based trading platforms already including relay/communication logic.
 
-## Running the Demo
-Berofe running the trader clients and the coordinator start a test mint using a fake funds source.
+## Testing
+The current `NostrClient` code only uses an in memory local test relay, which must be started before any tests.
+
+By now we use the relay at `git@github.com:rodant/bucket.git`. To start the relay locally:
+1. Checkout the master branch from the repo above.
+2. Run `yarn`, only needed the first time.

```

Comment:
May we should add yarn to the nix shell and nix flake dev envs? And /bucket to .gitignore so cloning it in the working dir doesnt lead to an accidental commit?

Reply:
Currently I just checkout bucket outside the repo. I'm not sure whether we will use bucket in the long term. I would explore first the possibility of an in-process relay for the tests, this would be even better. But some research is still required for that. Independently of the approach we choose, this is better done in a separated PR.